<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1255</width>
    <height>820</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1255</width>
    <height>820</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1255</width>
    <height>846</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>DejaVu Serif</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Streamer Par William Garneau</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnLecture">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>740</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Lecture</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spnVitesse">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>740</y>
      <width>48</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maximum">
     <number>3</number>
    </property>
    <property name="value">
     <number>1</number>
    </property>
   </widget>
   <widget class="QLabel" name="lblVitesse">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>745</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Vitesse de lecture</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="ckLoop">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>770</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Lecture en boucle</string>
    </property>
   </widget>
   <widget class="QSlider" name="slidVideo">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>720</y>
      <width>1151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1252</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblVideo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-5</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1280</width>
      <height>720</height>
     </size>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Lbl</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnQuit">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>770</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFS">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>740</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mettre en plein écran</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblConnected">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>750</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Nombre de clients connectés :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblClientCount">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>750</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblIP">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>770</y>
      <width>261</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1255</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="sbTimer"/>
  <action name="btnFullScreen">
   <property name="text">
    <string>Plein Écran</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
